src/convert_image/x86.rs

```diff
@@ -634,150 +929,50 @@ pub fn nv12_bt709_bgra_lrgb(
     )
 }
 
-pub fn rgb_lrgb_bgra_lrgb(
```

Comment:
Moving this function introduces a big diff that we could avoid like in sse2 and avx2